### PR TITLE
Audio upload to s3 bucket and Add url to telemetry logs

### PR DESCRIPTION
**Pull request by sudeeppr1998**

### CODE REVIEW

File: src/components/VoiceCompair/VoiceCompair.js [hunk 1]
```diff
@@ -211,8 +213,34 @@ const VoiceCompair = props => {
 
         let word_result = (result_per_words == 100) ? "correct" : "incorrect";
 
+        const client = new S3Client({
```
Comment (gouravmore) [inline src/components/VoiceCompair/VoiceCompair.js:216]: can we add this in one place? looks like it duplicated in - both components

File: src/components/VoiceCompair/VoiceCompair.js [hunk 2]
```diff
@@ -211,8 +213,34 @@ const VoiceCompair = props => {
...
+          region: process.env.REACT_APP_AWS_s3_region,
```
Comment (gouravmore) [inline src/components/VoiceCompair/VoiceCompair.js:217]: what if - we no need to push this data to s3?

File: src/components/VoiceCompair/VoiceCompair.js [hunk 3]
```diff
@@ -211,8 +213,34 @@ const VoiceCompair = props => {
...
+          const response = await client.send(command);
+          console.log(response);
+          console.log(`${process.env.REACT_APP_AWS_s3_BUCKET_URL}/${audioFileName}`);
+        } catch (err) {
+          console.error(err);
+        }
+
         response({ // Required
-            "target": localStorage.getItem('contentText'), // Required. Target of the response
+            "target": `${process.env.REACT_APP_AWS_s3_BUCKET_URL}/${audioFileName}`, // Required. Target of the response
```
Comment (gouravmore) [inline src/components/VoiceCompair/VoiceCompair.js:243]: if we do not want this data then?

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/components/VoiceCompair/VoiceCompair.js (hunk 3 above)
Comment (sudeeppr1998) [reply src/components/VoiceCompair/VoiceCompair.js]: Added condition in new commit

File: src/components/VoiceCompair/VoiceCompair.js (hunk 2 above)
Comment (sudeeppr1998) [reply src/components/VoiceCompair/VoiceCompair.js]: Added condition in new commit with new env variable

File: src/components/VoiceCompair/VoiceCompair.js (hunk 1 above)
Comment (sudeeppr1998) [reply src/components/VoiceCompair/VoiceCompair.js]: Added in config file

**Pull request merged**